packages/@aws-cdk/aws-mediastore/.npmignore

```diff
@@ -16,4 +16,7 @@ dist
 .LAST_PACKAGE
 .jsii
 
-*.tsbuildinfo
\ No newline at end of file
+*.tsbuildinfo
+
+# Include .jsii
+!.jsii
```

Comment:
Err... can you remove the `.jsii` line from L17 then?

Reply:
This got added by pkglint. I'm not going to fight it.